src/handlers/comment.ts

```diff
@@ -0,0 +1,58 @@
+import { isUserAdminOrBillingManager } from "../shared/issue";
+import { Context } from "../types/context";
+import { isCommentEvent } from "../types/typeguards";
+
+export async function setLabels(context: Context, body: string) {
+  const logger = context.logger;
+  if (!isCommentEvent(context)) {
+    return logger.debug("Not an comment event");
+  }
+
+  const payload = context.payload;
+  const sender = payload.sender.login;
+
+  const sufficientPrivileges = await isUserAdminOrBillingManager(context, sender);
+  if (!sufficientPrivileges) return logger.info(`You are not an admin and do not have the required permissions to access this function.`); // if sender is not admin, return
+
+  if (!payload.issue) return context.logger.info(`Skipping '/labels' because of no issue instance`);
+
+  if (body.startsWith("/labels")) {
```

Comment:
I guess we need to break this apart into a separate plugin or something right? The commands must be defined at the plugin config yml level

Reply:
Maybe we can have main command that's can be configurable in the config and sub-commands so `/labels access` and `/labels multiplier`

Reply:
Like a commands name space. That's an idea. I think we should aim to copy some other developer tooling with a similar implementation, but for now this seems to make sense.